components/automate-ui/package.json

```diff
@@ -46,6 +46,7 @@
     "d3": "^5.15.0",
     "diff2html": "^2.12.2",
     "file-saver": "^2.0.2",
+    "fp-ts": "^2.5.3",
```

Comment:
If you are modifying package.json, you must also include package-lock.json.

Reply:
actually my  `package.json` have lots of local changes so I have not added this file.

Reply:
Hmm... ok, after this merges if you still have other changes mixed in, I can regenerate the specific package-lock.json changes and commit those.

Reply:
sure, Thanks :)